Sources/StreamChat/Extensions/URLRequest+cURL.swift

```diff
@@ -32,7 +32,9 @@ extension URLRequest {
                 .replacingOccurrences(of: "\"", with: "\\\"")
             cURL.append("-d \"\(escapedBody)\"")
         }
-        cURL.append("\"\(url.absoluteString)\"")
+        let urlString = url.absoluteString
+            .replacingOccurrences(of: "$", with: "%24") // encoded JSON payload
+        cURL.append("\"\(urlString)\"")
```

Comment:
curl description did not work for channel list queries where we have JSON for the payload parameter in the URL. This is only for debugging and does not affect the actual request.